### PR TITLE
[DowngradePhp80] Allow FuncCall on DowngradeNamedArgumentRector

### PR DESCRIPTION
**Pull request by samsonasik**
Fixes https://github.com/rectorphp/rector/issues/6547 . The Logic to re-fill value need to be updated as well for jumping key fill named arguments.

### CODE REVIEW

**Comment by samsonasik** [comment]
Fixed 🎉 need to clean up next.

**Comment by samsonasik** [comment]
@TomasVotruba using native `ReflectionFunction` to get default value seems got real value, any way to convert real value to `Expr` node? Using `ConstFetch(new Name())` seems not the way to go for Boolean value, as `true` will be casted to `1`

**Comment by samsonasik** [comment]
I got it :tada: , I can create new PHPStan\Reflection\Php\PhpParameterReflection from ReflectionParameter object.

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba it is ready for review.

File: rules/DowngradePhp80/NodeAnalyzer/UnnamedArgumentResolver.php [hunk 1]
```diff
@@ -54,66 +106,62 @@ public function resolveFromReflection(
...
-    }
-
     /**
      * @param Arg[] $unnamedArgs
+     * @param ParameterReflection[]|PhpParameterReflection[] $parameters
      * @return Arg[]
      */
-    private function fillArgValues(
-        int $highestParameterPosition,
+    private function fillFromJumpedNamedArgs(
```
Comment (TomasVotruba) [inline rules/DowngradePhp80/NodeAnalyzer/UnnamedArgumentResolver.php]: It seems there is quiote a lot going on in this service now.
Could you extract this logic to an external service?

File: rules/DowngradePhp80/Rector/MethodCall/DowngradeNamedArgumentRector.php [hunk 2]
```diff
@@ -75,7 +77,7 @@ private function execute($a = null, $b = null)
     }
 
     /**
-     * @param MethodCall|StaticCall $node
+     * @param MethodCall|StaticCall|New_|FuncCall $node
```
Comment (TomasVotruba) [inline rules/DowngradePhp80/Rector/MethodCall/DowngradeNamedArgumentRector.php:80]: :+1:

**Review by TomasVotruba — COMMENTED** [review]
Thank you, I've left few comments

File: rules/DowngradePhp80/NodeAnalyzer/UnnamedArgumentResolver.php (hunk 1 above)
Comment (samsonasik) [reply rules/DowngradePhp80/NodeAnalyzer/UnnamedArgumentResolver.php]: implemented https://github.com/rectorphp/rector-src/pull/466/commits/4c4b9308822c33c0aea8161516a8229510c16a46

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba I think it is ready.

**Comment by TomasVotruba** [comment]
Thanks :clap:

**Pull request merged**